src/Features/Core/Portable/DesignerAttribute/AbstractDesignerAttributeIncrementalAnalyzer.cs

```diff
@@ -4,73 +4,79 @@
 
```

Comment:
type is renamed, but i left in the same file as otherwise the diff doesn't understand.